File: Java Script - Introduction to Programming/6.2. Drawing on the Console - Exam Problems/03. Stop.js
function stop([n]) {
  n = +n
  //draw top  
  console.log('.'.repeat(n + 1) + '_'.repeat(n * 2 + 1) + '.'.repeat(n + 1))
  //draw up part
  let add = 1
  for (i = 1; i <= n; i++) {
    console.log('.'.repeat((n + 1) - i) + '//' + '_'.repeat((((n + 1) * 2) - 4) + add) + '\\\\' + '.'.repeat((n + 1) - i))
    add += 2
  }
  //draw sing Stop
  if (n < 4) {
    console.log('//' + '_'.repeat(n) + 'STOP!' + '_'.repeat(n) + '\\\\')
  } else {
    console.log('//' + '_'.repeat(((n + n) * 2 - 5) / 2) + 'STOP!' + '_'.repeat(((n + n) * 2 - 5) / 2) + '\\\\')
    add += 2;
  };
  //draw bottom
  add = 1
  for (i = n; i >= 1; i--) {
    console.log('.'.repeat((n) - i) + '\\\\' + '_'.repeat((((n + n) * 2)) - add) + '//' + '.'.repeat((n) - i))
    add += 2
  }
}

stop([7])

File: Pyrhon - Introduction to Programming/6.2. Drawing on the Console - Exam Problems/03. Stop.py
n = int(input())
#draw top  
print('.' * (n + 1) + '_' *(n * 2 + 1) + '.' *(n + 1))
#draw up part
add = 1
for i in range(n):
    print('.' * (n - i) + '//' + '_' * ((((n + 1) * 2) - 4) + add) + '\\\\' + '.' *(n  - i))
    add += 2

#draw sing Stop
if n < 4:
    print('//' + '_' * n + 'STOP!' + '_' * n + '\\\\')
else:
    print('//' + '_' * (((n + n) * 2 - 5) // 2) + 'STOP!' + '_' * (((n + n) * 2 - 5) // 2) + '\\\\')

#draw bottom
add = 1
for i in range(n,0,-1):
    print('.' * (n - i) + '\\\\' + '_' * (((n + n) * 2) - add) + '//' + '.' * (n - i))
    add += 2

